x-pack/test/security_solution_endpoint/apps/endpoint/endpoint_list.ts

```diff
@@ -77,54 +78,61 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
+    it('does not show the details flyout initially', async () => {
       await testSubjects.missingOrFail('hostDetailsFlyout');
     });
 
-    it('display details flyout when the hostname is clicked on', async () => {
-      await (await testSubjects.find('hostnameCellLink')).click();
-      await testSubjects.existOrFail('hostDetailsUpperList');
-      await testSubjects.existOrFail('hostDetailsLowerList');
-    });
+    describe('when the hostname is clicked on,', () => {
+      it('display the details flyout', async () => {
+        await (await testSubjects.find('hostnameCellLink')).click();
+        await testSubjects.existOrFail('hostDetailsUpperList');
+        await testSubjects.existOrFail('hostDetailsLowerList');
+      });
 
-    it('update details flyout when new hostname is clicked on', async () => {
-      // display flyout for the first host in the list
-      await (await testSubjects.findAll('hostnameCellLink'))[0].click();
-      await testSubjects.existOrFail('hostDetailsFlyoutTitle');
-      const hostDetailTitle0 = await testSubjects.getVisibleText('hostDetailsFlyoutTitle');
-      // select the 2nd host in the host list
-      await (await testSubjects.findAll('hostnameCellLink'))[1].click();
-      await pageObjects.endpoint.waitForVisibleTextToChange(
-        'hostDetailsFlyoutTitle',
-        hostDetailTitle0
-      );
-      const hostDetailTitle1 = await testSubjects.getVisibleText('hostDetailsFlyoutTitle');
-      expect(hostDetailTitle1).to.not.eql(hostDetailTitle0);
-    });
+      it('updates the details flyout when a new hostname is selected from the list', async () => {
+        // display flyout for the first host in the list
+        await (await testSubjects.findAll('hostnameCellLink'))[0].click();
+        await testSubjects.existOrFail('hostDetailsFlyoutTitle');
+        const hostDetailTitle0 = await testSubjects.getVisibleText('hostDetailsFlyoutTitle');
+        // select the 2nd host in the host list
+        await (await testSubjects.findAll('hostnameCellLink'))[1].click();
+        await pageObjects.endpoint.waitForVisibleTextToChange(
+          'hostDetailsFlyoutTitle',
+          hostDetailTitle0
+        );
+        const hostDetailTitle1 = await testSubjects.getVisibleText('hostDetailsFlyoutTitle');
+        expect(hostDetailTitle1).to.not.eql(hostDetailTitle0);
+      });
+
+      it('has the same flyout info when the same hostname is selected', async () => {
+        // display flyout for the first host in the list
+        await (await testSubjects.findAll('hostnameCellLink'))[1].click();
+        await testSubjects.existOrFail('hostDetailsFlyoutTitle');
+        const hostDetailTitleInitial = await testSubjects.getVisibleText('hostDetailsFlyoutTitle');
+        // select the same host in the host list
+        await (await testSubjects.findAll('hostnameCellLink'))[1].click();
+        await sleep(500); // give page time to refresh and verify it did not change
```

Comment:
this was being skipped, before right?  `sleep()` in tests make me a bit nervous, just wanna make sure this is run enough times to ensure this isn't flaky